com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/LaunchRequestHandler.java

```diff
@@ -55,10 +55,10 @@
     @Override
     public void handle(Command command, Arguments arguments, Response response, IDebugAdapterContext context) {
         LaunchArguments launchArguments = (LaunchArguments) arguments;
-        if (StringUtils.isBlank(launchArguments.mainClass) || launchArguments.classPaths == null
-                || launchArguments.classPaths.length == 0) {
+        if (StringUtils.isBlank(launchArguments.mainClass)
+                || (isEmptyArray(launchArguments.modulePaths) && isEmptyArray(launchArguments.classPaths))) {
             AdapterUtils.setErrorResponse(response, ErrorCode.ARGUMENT_MISSING,
-                    String.format("Failed to launch debuggee VM. Missing mainClass or classPath options in launch configuration"));
+                    String.format("Failed to launch debuggee VM. Missing mainClass or modulePaths/classPaths options in launch configuration"));
```

Comment:
"Failed to launch debuggee VM. Missing mainClass or modulePaths/classPaths options in launch configuration"
==> 
"Failed to launch debuggee VM. Missing mainClass or modulePaths/classPaths options in `the` launch configuration"

Reply:
fixed.